index.html

```diff
@@ -3490,12 +3490,12 @@ <h4>JSON Web Token Extensions</h4>
               <li>
 <code>vc</code>: JSON object, which MUST be present in a JWT
 <a>verifiable credential</a>. The object contains the
-<a>verifiable credential</a> according to this specification.
+<a>credential</a> according to this specification.
               </li>
               <li>
 <code>vp</code>: JSON object, which MUST be present in a JWT
 <a>verifiable presentation</a>. The object contains the
-<a>verifiable presentation</a> according to this specification.
```

Comment:
This change should be accepted for the reasons I gave in https://github.com/w3c/vc-data-model/pull/828/files#r753698782